src/OpenSage.Game/Content/ContentManager.cs

```diff
@@ -79,6 +79,11 @@ public sealed class ContentManager : DisposableBase
 
             IniDataContext.LoadIniFiles(@"Data\INI\Default\Object.ini");
             IniDataContext.LoadIniFiles(@"Data\INI\Object");
+            IniDataContext.LoadIniFiles(@"Data\INI\multiplayer.ini");
+
+            //german2 has an Maps folder - english has an maps folder
+            //so we need to deal with different spells in different versions
+            IniDataContext.LoadIniFile(@"Maps\MapCache.ini", @"maps\MapCache.ini");
```

Comment:
I don't think is the right approach here. The entire filesystem should probably be case insensitive, because windows doesn't care about that anyways. So maybe the GetFile method in FileSystem.cs should take care of that. Need opinions of @tgjones and @paavohuhtala here.

Reply:
I just checked again. This *SHOULD* already work. The filesystem is case insensitive.